src/AWS.Deploy.Orchestration/ServiceHandlers/AWSElasticBeanstalkHandler.cs

```diff
@@ -83,7 +144,21 @@ public List<ConfigurationOptionSetting> GetEnvironmentConfigurationSettings(Reco
         {
             var additionalSettings = new List<ConfigurationOptionSetting>();
 
-            foreach (var tuple in Constants.ElasticBeanstalk.OptionSettingQueryList)
+            List<(string OptionSettingId, string OptionSettingNameSpace, string OptionSettingName)> tupleList;
+            switch (recommendation.Recipe.Id)
```

Comment:
Minor: consider moving the switch logic to a separate function to improve the readability of GetEnvironmentConfigurationSettings

Reply:
The function does not have enough logic as of today to move it to a separate function.